cmd/cli/root.go

```diff
@@ -8,24 +8,36 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/bacalhau-project/bacalhau/pkg/config"
-	"github.com/bacalhau-project/bacalhau/pkg/logger"
-	"github.com/bacalhau-project/bacalhau/pkg/system"
-	"github.com/bacalhau-project/bacalhau/pkg/telemetry"
-	"github.com/bacalhau-project/bacalhau/pkg/version"
 	"github.com/rs/zerolog/log"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 	"go.opentelemetry.io/otel/trace"
+
+	"github.com/bacalhau-project/bacalhau/cmd/cli/cancel"
+	"github.com/bacalhau-project/bacalhau/cmd/cli/create"
+	"github.com/bacalhau-project/bacalhau/cmd/cli/describe"
+	"github.com/bacalhau-project/bacalhau/cmd/cli/devstack"
+	"github.com/bacalhau-project/bacalhau/cmd/cli/docker"
+	"github.com/bacalhau-project/bacalhau/cmd/cli/get"
+	"github.com/bacalhau-project/bacalhau/cmd/cli/id"
+	"github.com/bacalhau-project/bacalhau/cmd/cli/list"
+	"github.com/bacalhau-project/bacalhau/cmd/cli/logs"
+	"github.com/bacalhau-project/bacalhau/cmd/cli/serve"
+	"github.com/bacalhau-project/bacalhau/cmd/cli/simulate"
+	"github.com/bacalhau-project/bacalhau/cmd/cli/validate"
+	version2 "github.com/bacalhau-project/bacalhau/cmd/cli/version"
```

Comment:
why `version2`?

Reply:
my IDE did it when moving files around, will remove the alias so this is imported as `version`.